mission_control/navigator_missions/nav_missions/circle_tower.py

```diff
@@ -39,7 +45,7 @@ def main(navigator, **kwargs):
     # TODO: Check which side to go to based on relative distance
     center = navigator.move.set_position(buoy_field_point).set_orientation(get_solar_q())
     obs_point = center.backward(BF_WIDTH / 2 + BF_EST_COFIDENCE) 
-    yield obs_point.go()
+    #yield obs_point.go()
```

Comment:
Lot's of commented out code here since we will want to test the main functionality first before testing the whole mission